data_updates/00010_20230427-151329_users.py

```diff
@@ -22,25 +22,30 @@ def forwards(self, mongodb_collection, _mongodb_database):
 
         for user in mongodb_collection.find({}):
             company = companies.get(user.get("company"))
-            if not company:
-                continue
-
-            company_section_names = get_company_section_names(company)
-            company_product_ids = get_company_product_ids(company)
-
-            updates = {
-                "sections": {
-                    section: enabled and section in company_section_names
-                    for section, enabled in (user.get("sections") or {}).items()
-                },
-                "products": [
-                    product
-                    for product in user.get("products") or []
-                    if product.get("section") in company_section_names and product.get("_id") in company_product_ids
-                ],
-            }
-
-            print(mongodb_collection.update({config.ID_FIELD: user.get(config.ID_FIELD)}, {"$set": updates}))
+            updates = {}
+
+            if isinstance(user.get("sections"), list):
+                updates["sections"] = {name: True for name in user["sections"]}
+
+            if company:
+                company_section_names = get_company_section_names(company)
+                company_product_ids = get_company_product_ids(company)
+                user_sections = updates.get("sections") or user.get("sections") or {}
+
+                updates = {
```

Comment:
This will override the `sections` attribute changes from above, will need to use `update` instead.